eng/pipelines/common/xplat-setup.yml

```diff
@@ -50,6 +50,13 @@ jobs:
         - name: ROOTFS_DIR
           value: ${{ parameters.jobParameters.crossrootfsDir }}
 
+      - ${{ if ne(parameters.jobParameters.runtimeFlavor, '') }}:
```

Comment:
runtimeFlavor should always be passed down as a `jobParameter` since platform-matrix is passing it down always, so no need to have this empty check any more.